#' @title Error handling
#'
#' @description Used to handle errors internally generated by functions without breaking the function
#`
#' @param code code to be tested to handle errors
#' @return error message
#' @examples
#' x=matrix(runif(10,0,1),ncol=2)
#' is.err(x[,4])
#' @export
is.err <- function(code) {
  tryCatch(code,
           error = function(c) TRUE,
           warning = function(c) TRUE,
           message = function(c) FALSE
  )
}
